### PR TITLE
doc/cephadm: add steps to the adoption process

### PR DESCRIPTION
**Pull request by gurubert**
There are a few steps missing the get the full orchestrator experience.

Signed-off-by: Robert Sander <gurubert@gurubert.de>

### CODE REVIEW

File: doc/cephadm/adoption.rst [hunk 1]
```diff
@@ -211,3 +219,22 @@ Adoption process
 
 #. Check the output of the command ``ceph health detail`` for cephadm warnings
    about stray cluster daemons or hosts that are not yet managed by cephadm.
+
+#. Remove Ceph packages from the host except for ceph-common to keep Ceph commandline tools.
```
Comment (anthonyeleven) [inline doc/cephadm/adoption.rst:223]: ceph-common in double backticks?

**Review by anthonyeleven — CHANGES_REQUESTED** [review]

File: doc/cephadm/adoption.rst (hunk 1 above)
Comment (gurubert) [reply doc/cephadm/adoption.rst]: added backticks in commit 08384dbe7f067404523191fbf32c5a91e7241ec1

**Review by anthonyeleven — APPROVED** [review]

File: doc/cephadm/adoption.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/cephadm/adoption.rst]: Danke.  The test is now failing because that commit isn't signed-off.  Maybe squash the two?

**Comment by adk3798** [comment]
@gurubert the signed-off-by check is complaining about the commit title

```
"The title/s of following commit/s is/are not started with 'doc', but they only touch files under 'doc/'. Please make sure the commit titles",
                "are started with 'doc'. See the 'Submitting Patches' guide:",
```

**Comment by anthonyeleven** [comment]
> @gurubert the signed-off-by check is complaining about the commit title
> 
> ```
> "The title/s of following commit/s is/are not started with 'doc', but they only touch files under 'doc/'. Please make sure the commit titles",
>                 "are started with 'doc'. See the 'Submitting Patches' guide:",
> ```

Please amend the title of the second commit and push to fix this.

**Comment by zdover23** [comment]
@gurubert: I'm pinging you to ask you to squash the commits as described here: https://docs.ceph.com/en/quincy/start/documenting-ceph/#squash-extraneous-commits

If you haven't squashed and pushed by next week, I'll raise another PR that will ingest the changes made in this PR into the body of the documentation. I will credit you and reference this PR in that PR, if it comes to that.

Thank you for this change.

**Pull request closed without merging**